modules/kernel/pom.xml

```diff
@@ -225,6 +225,14 @@
                     </execution>
                 </executions>
             </plugin>
+            <plugin>
```

Comment:
Would it be better if we add this to axis2-parent pom instead, since it is the parent pom for all modules?